### DISCUSSION TITLE
How to deploy a Spring Boot project on cloud-app?

### DISCUSSION THREAD

**Discussion by NeoCarm** [apache/camel-karavan#761] [Q&A] [answered]
I created a project on cloud-app and added an application.properties file and a YAML file. What else do I need to deploy a Spring Boot project and successfully run a Camel Route on cloud-app?

**Comment by NeoCarm** [comment]
Another question, when I commit to the git repository on cloud-app, cloud-app will delete all the content that is not there.

**Comment by mgubaidullin** [comment, marked as answer]
Deploy project
1. Commit
2. Got to operations tab and click deploy

When you commit to the git repository on cloud-app, cloud-app will rewrite `PROJECT_NAME` folder
